packages/flutter_tools/lib/src/commands/drive.dart

```diff
@@ -244,7 +244,10 @@ class DriveCommand extends RunCommandBase {
         throwToolExit('Application failed to start. Will not run test. Quitting.', exitCode: 1);
       }
       observatoryUri = result.observatoryUri.toString();
-      await device.dds.startDartDevelopmentService(Uri.parse(observatoryUri), ipv6);
+      // TODO(bkonyi): add web support
```

Comment:
Could you link this to an issue?

Reply:
Done, but for convenience here it is: https://github.com/flutter/flutter/issues/61259